packages/components/src/components/Checkbox/ControlledCheckbox/ControlledCheckbox.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Control, Controller } from "react-hook-form";
+import type { ControllerProps } from "react-hook-form";
+import { CheckedState } from "@radix-ui/react-checkbox";
+import { Checkbox } from "../Checkbox";
+import type { CheckboxProps } from "../Checkbox";
+
+export interface ControlledCheckboxProps
+  extends Pick<ControllerProps, "name" | "rules">,
+    Omit<CheckboxProps, "checked" | "error" | "name" | "onBlur" | "value"> {
+  // Invoked with `useForm`. Set to any to allow `any` number of form inputs.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  control?: Control<any>;
```

Comment:
out of curiosity: can/should be `unknown` here?

Reply:
It doesn't like being set to unknown 😞 
<img width="463" alt="image" src="https://user-images.githubusercontent.com/3478163/229899603-4fdca230-5dc5-462d-984b-156d65f6a6c1.png">